### ISSUE TITLE
Compatibility with Ruby 2.7

### ISSUE THREAD

**Issue by pedrofurtado** [flyerhzm/bullet#485]
@flyerhzm Using the gem with Ruby 2.7, gives me the following deprecation:

```bash
/usr/local/rvm/gems/ruby-2.7.0@myapp/gems/bullet-6.0.2/lib/bullet.rb:41: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

**Comment by connorshea** [comment]
I believe this is fixed on master by a7accb25608ba8a51ca271fa55771576a79012e6, so we just need a new release.

**Comment by flyerhzm** [comment]
yes, I just released bullet 6.1.0, which fixed this warning.
